### DISCUSSION TITLE
Allow for the visualization of small graphs using Graphviz

### DISCUSSION THREAD

**Discussion by angelip2303** [rudof-project/rudof#213] [Ideas]
It would be cool to support an export format in Graphviz.

**Comment by labra** [comment]
Indeed, this functionality is a very nice feature to have. 

For simple graphs, we can get inspiration from [the implementation that we already did in Scala](https://github.com/weso/shaclex/tree/master/modules/sgraph/src/main/scala/es.weso.rdf.sgraph) and it was quite easy to do. In that implementation, we defined an intermediate data structure called `SGraph` (for simple graph) and we converted RDF graphs to SGraph, and those SGraphs could be easily serialized to dot format.  

This tools is used by [rdfshape](https://rdfshape.weso.es/link/17310727911)
